ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-service-insertion.cfg

```diff
@@ -5,6 +5,20 @@ transceiver qsfp default-mode 4x10G
 hostname router-service-insertion
 !
 router service-insertion
+   connection connection1
+      interface Ethernet2/2.2 next-hop 10.10.10.10
```

Comment:
make one intefrace not a subinterface in your examples.

Reply:
Added one example with interface.